app/src/main/res/values/strings.xml

```diff
@@ -197,11 +197,17 @@ Tap this message (or hit back) to skip this step.</string>
   <string name="navigation_item_feedback">Feedback</string>
   <string name="navigation_item_logout">Logout</string>
   <string name="navigation_item_info">Tutorial</string>
+  <string name="navigation_item_developer_plans">Developer plans</string>
 
   <string name="nearby_needs_permissions">Nearby places cannot be displayed without location permissions</string>
   <string name="no_description_found">no description found</string>
   <string name="nearby_info_menu_commons_article">Commons file page</string>
   <string name="nearby_info_menu_wikidata_article">Wikidata item</string>
-    <string name="error_while_cache">Error while caching pictures</string>
+  <string name="error_while_cache">Error while caching pictures</string>
+  <string name="feedback_popup_title">Feedback wanted</string>
+  <string name="feedback_popup_description">We are planning several new features and improvements for the app! Would you like to review them and provide feedback? (You can always access this by selecting "Developer plans" in the navigation drawer)</string>
+  <string name="feedback_popup_decline">No thanks</string>
+  <string name="feedback_popup_accept">Sure, take me there!</string>
+  <string name="feedback_page_url">https://meta.wikimedia.org/wiki/Grants:Project/Improve_\'Upload_to_Commons\'_Android_App/Renewal#Endorsements</string>
```

Comment:
I think we should probably not link directly to the Endorsements section, since people have to read it first. :) I would just take them to `https://meta.wikimedia.org/wiki/Grants:Project/Improve_\'Upload_to_Commons\'_Android_App/Renewal` and remove the `#Endorsements` personally